resources/js/Pages/Admin/Dashboard/Index.tsx

```diff
@@ -28,3 +28,26 @@ export default function Index({ message, button }: Props) {
         </div>
     );
 }
+*/}
```

Comment:
I think we could probably delete this sample form now that you've made 2 real forms! 🎉